core/trino-main/src/main/java/io/trino/sql/DynamicFilters.java

```diff
@@ -191,12 +191,16 @@ public static Optional<Descriptor> getDescriptor(Expression expression)
         return Optional.of(new Descriptor(new DynamicFilterId(id), probeSymbol, operator, nullAllowed));
     }
 
-    private static boolean isDynamicFilterFunction(FunctionCall functionCall)
+    public static boolean isDynamicFilterFunction(CatalogSchemaFunctionName functionName)
     {
-        CatalogSchemaFunctionName functionName = ResolvedFunction.extractFunctionName(functionCall.getName());
         return functionName.equals(builtinFunctionName(Function.NAME)) || functionName.equals(builtinFunctionName(NullableFunction.NAME));
     }
 
+    private static boolean isDynamicFilterFunction(FunctionCall functionCall)
```

Comment:
move it above `public static boolean isDynamicFilterFunction(CatalogSchemaFunctionName functionName`